definitions.json

```diff
@@ -56,7 +56,7 @@
             "properties": {
                 "type_id": { "$ref": "#/definitions/renderer" },
                 "confidence": {
-                    "description": "Estimated percent confidence in the correctness of the rendering.",
+                    "description": "DEPRECATED: Estimated percent confidence in the correctness of the rendering.",
```

Comment:
We don't currently do anything with this, so agree it is important that it be flagged since it could otherwise be confusing, but I suspect we may return to this at some point, so I view it more as a placeholder than something that we did at one point and are removing forever. Suggest:

"DEPRECATED (but reserved for future use): Estimated..."